### DISCUSSION TITLE
conda-forge packaging

### DISCUSSION THREAD

**Discussion by roaldarbol** [abhiTronix/deffcode#54] [General]
Hi @abhiTronix!
I'm in the process of packaging up a bigger project ([FreeMoCap](https://github.com/freemocap/freemocap)) on conda-forge, and one of their packages depends on `deffcode`. Would you be fine with me (trying to) making and maintaining a recipe on conda-forge? I would be happy to add you to it too, but don't want to cause any extra work for you. :-)
Cheers,
Mikkel

**Comment by roaldarbol** [comment]
I've started making the recipe in this PR: https://github.com/conda-forge/staged-recipes/pull/28185

**Comment by abhiTronix** [comment]
Sure.

**Comment by abhiTronix** [comment]
> I would be happy to add you to it too, but don't want to cause any extra work for you. :-)

@roaldarbol that sounds fantastic to me. Let me know if any help required form my end.

**Reply by roaldarbol** [reply]
Great! The recipe seems to have cleared all checks, so should go through soon! I'll update you. 😃

**Reply by roaldarbol** [reply]
It has now been approved, and the feedstock lives here: https://github.com/conda-forge/deffcode-feedstock
It should be available to download with conda from conda-forge before long!

**Reply by roaldarbol** [reply]
It's already available! https://anaconda.org/conda-forge/deffcode
